project/.php_cs.dist.twig

```diff
@@ -33,33 +33,25 @@ $rules = [
     'ordered_class_elements' => true,
     'ordered_imports' => true,
     'phpdoc_order' => true,
+    '@PHP56Migration' => true,
+    '@PHP56Migration:risky' => true,
+    '@PHPUnit57Migration:risky' => true,
+{% if php|first in ['7.0', '7.1'] %}
+    '@PHP70Migration' => true,
+    '@PHP70Migration:risky' => true,
+    '@PHPUnit60Migration:risky' => true,
+{% endif %}
+{% if php|first == '7.1' %}
+    '@PHP71Migration' => true,
+    '@PHP71Migration:risky' => true,
+    'compact_nullable_typehint' => true,
+{% endif %}
     // To be tested before insertion:
 //    'strict_comparison' => true,
 //    'strict_param' => true,
 //    'php_unit_strict' => true,
 ];
 
-if (\PHP_VERSION_ID >= 50600) {
-    $rules = array_merge($rules, [
-        '@PHP56Migration' => true,
-        '@PHP56Migration:risky' => true,
-        '@PHPUnit57Migration:risky' => true,
-    ]);
-}
-if (\PHP_VERSION_ID >= 70000) {
-    $rules = array_merge($rules, [
-        '@PHP70Migration' => true,
-        '@PHP70Migration:risky' => true,
-        '@PHPUnit60Migration:risky' => true,
-    ]);
-}
-if (\PHP_VERSION_ID >= 70100) {
-    $rules = array_merge($rules, [
-        '@PHP71Migration' => true,
-        '@PHP71Migration:risky' => true,
-        'compact_nullable_typehint' => true,
-    ]);
-}
 
 $finder = PhpCsFixer\Finder::create()
     ->in(preg_match('/dev-kit\/project$/', __DIR__, $matches) ? __DIR__.'/..' : __DIR__)
```

Comment:
This trick is no longer needed if we do it this way. 

Same for the dev-kit configuration